gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/application/use_case/ImportApplicationCRDUseCase.java

```diff
@@ -51,30 +67,41 @@ public class ImportApplicationCRDUseCase {
     private final ImportApplicationCRDDomainService importApplicationCRDDomainService;
     private final ApplicationMetadataCrudService applicationMetadataCrudService;
     private final ApplicationMetadataQueryService applicationMetadataQueryService;
+    private final MemberQueryService memberQueryService;
+    private final UserDomainService userDomainService;
 
     public ImportApplicationCRDUseCase(
         ApplicationCrudService applicationCrudService,
         ImportApplicationCRDDomainService importApplicationCRDDomainService,
         ApplicationMetadataCrudService applicationMetadataCrudService,
-        ApplicationMetadataQueryService applicationMetadataQueryService
+        ApplicationMetadataQueryService applicationMetadataQueryService,
+        MemberQueryService memberQueryService,
+        UserDomainService userDomainService
     ) {
         this.applicationCrudService = applicationCrudService;
         this.importApplicationCRDDomainService = importApplicationCRDDomainService;
         this.applicationMetadataCrudService = applicationMetadataCrudService;
         this.applicationMetadataQueryService = applicationMetadataQueryService;
+        this.memberQueryService = memberQueryService;
+        this.userDomainService = userDomainService;
     }
 
     public record Output(ApplicationCRDStatus status) {}
 
     public record Input(AuditInfo auditInfo, ApplicationCRDSpec crd) {}
 
     public Output execute(Input input) {
+        BaseApplicationEntity application;
```

Comment:
shall we use var to match what can be found in other classes of this new package ? (assigning in the try catch)